web_src/js/index.js

```diff
@@ -3002,7 +3002,7 @@ window.initHeatmap = function (appElementId, heatmapUser, locale) {
     methods: {
       loadHeatmap(userName) {
         const self = this;
-        $.get(`${this.suburl}/api/v1/users/${userName}/heatmap`, (chartRawData) => {
+        $.get(`${this.suburl}/api/v1/users/${encodeURI(userName)}/heatmap`, (chartRawData) => {
```

Comment:
```suggestion
        $.get(`${this.suburl}/api/v1/users/${encodeURIComponent(userName)}/heatmap`, (chartRawData) => {
```

`encodeURI` is only meant for full `URI`s.